src/DynamicRupture/FrictionLaws/BaseFrictionLaw.h

```diff
@@ -40,28 +40,27 @@ class BaseFrictionLaw : public FrictionSolver {
       SCOREP_USER_REGION_BEGIN(
           myRegionHandle, "computeDynamicRupturePrecomputeStress", SCOREP_USER_REGION_TYPE_COMMON)
       LIKWID_MARKER_START("computeDynamicRupturePrecomputeStress");
-      /// TODO: convert from strain to stress
+
       alignas(PAGESIZE_STACK)
           real qStressInterpolatedPlus[CONVERGENCE_ORDER][seissol::tensor::QInterpolated::size()] =
               {{0.0}};
       alignas(PAGESIZE_STACK)
           real qStressInterpolatedMinus[CONVERGENCE_ORDER][seissol::tensor::QInterpolated::size()] =
               {{0.0}};
+#ifdef USE_DAMAGEDELASTIC
+      // TODO: convert from strain to stress
 
       using QInterpolatedShapeT =
           const real(*)[seissol::dr::misc::numQuantities][seissol::dr::misc::numPaddedPoints];
       using QStressInterpolatedShapeT =
           real(*)[seissol::dr::misc::numQuantities][seissol::dr::misc::numPaddedPoints];
+
+      using namespace seissol::dr::misc::quantity_indices;
```

Comment:
Can we do it without it or is it called in too many places? Not a fan of using namespaces like this as it gets confusing when we are using other namespace calls in the same file.

Reply:
I agree with you in general, but I guess here it's just convenient to write `array[Q]` instead of `array[seissol::dr::misc::quantity_indices::Q]`...